alerts/apps_alerts.libsonnet

```diff
@@ -128,6 +128,19 @@
             },
             'for': '15m',
           },
+          {
+            expr: |||
+              sum by (namespace, pod, container) (kube_pod_container_status_waiting_reason{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s} * on(namespace, pod) group_left(owner_kind) kube_pod_owner)  > 0
```

Comment:
can you explain why we join the `kube_pod_owners` onto the waiting status here?

Reply:
@brancz sorry my bad, removed it b3fd3c643b050b0add8002a9be345a8149fe9ecb.

Reply:
sorry my bad, removed the `kube_pod_owners` join b3fd3c643b050b0add8002a9be345a8149fe9ecb.